apiserver/facades/client/modelconfig/service.go

```diff
@@ -19,6 +20,14 @@ type ModelConfigService interface {
 	UpdateModelConfig(context.Context, map[string]any, []string, ...config.Validator) error
 }
 
+// ModelSericve is an interface for interacting with model service.
```

Comment:
Same comment here about consistency. 
Also below and elsewhere in the PR  `current model's ...`